kext/sysent.c

```diff
@@ -254,7 +255,7 @@ bruteforce_sysent(mach_vm_address_t *out_kernel_base)
     *out_kernel_base = kernel_base;
     uint64_t segment_address = 0;
     uint64_t segment_size = 0;
-	if (version_major >= SIERRA) {
+	if (version_major >= MOJAVE) {
```

Comment:
This changes looks wrong to me, it must stay as before for onyx to remain working on 10.12 and 10.13.